src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/Az.RecoveryServices.SiteRecovery.psd1

```diff
@@ -1,232 +0,0 @@
-#
-# Module manifest for module 'Az.RecoveryServices.SiteRecovery'
-#
-# Generated by: Microsoft Corporation
-#
-# Generated on: 3/13/2018
-#
-
-@{
-
-# Script module or binary module file associated with this manifest.
-# RootModule = ''
-
-# Version number of this module.
-ModuleVersion = '0.2.2'
-
-# Supported PSEditions
-CompatiblePSEditions = 'Core', 'Desktop'
-
-# ID used to uniquely identify this module
-GUID = '8984d8d5-a901-4048-8f5f-1b1542cb6c9c'
-
-# Author of this module
-Author = 'Microsoft Corporation'
-
-# Company or vendor of this module
-CompanyName = 'Microsoft Corporation'
-
-# Copyright statement for this module
-Copyright = 'Microsoft Corporation. All rights reserved.'
-
-# Description of the functionality provided by this module
-Description = 'Microsoft Azure PowerShell - SiteRecovery service cmdlets for Azure Resource Manager in PowerShell and PowerShell Core'
-
-# Minimum version of the Windows PowerShell engine required by this module
-PowerShellVersion = '5.1'
-
-# Name of the Windows PowerShell host required by this module
-# PowerShellHostName = ''
-
-# Minimum version of the Windows PowerShell host required by this module
-# PowerShellHostVersion = ''
-
-# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-DotNetFrameworkVersion = '4.7.2'
-
-# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-# CLRVersion = ''
-
-# Processor architecture (None, X86, Amd64) required by this module
-# ProcessorArchitecture = ''
-
-# Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.2.2'; })
-
-# Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\AutoMapper.dll', 
```

Comment:
Are these assemblies in the Az.RecoveryServices RequiredAssemblies?

Reply:
Added.